README.md

```diff
@@ -0,0 +1,41 @@
+# Picover
+
+App for documenting memories from Android Guild meetings and parties
+
+## Contribution
```

Comment:
alternatively, we could use details and summary tag, more info [here]((https://gist.github.com/scmx/eca72d44afee0113ceb0349dd54a84a2))